.github/actions/run-publish-vr-screenshot/action.yml

```diff
@@ -0,0 +1,132 @@
+name: 'Run and Publish VR Screenshot'
```

Comment:
migration of https://github.com/microsoft/fluentui/blob/master/.devops/templates/runpublishvrscreenshot.yml which also mitigates duplication - no need to have separate baseline pipeline https://github.com/microsoft/fluentui/blob/master/azure-pipelines.vrt-baseline.yml